### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by watho**
Updated Readme with new lineart-option and Python2 workaround for non-latin characters.

### CODE REVIEW

**Comment by jcrowgey** [comment]
Thanks for this!

Two comments:

1) I'm pretty sure (but I could be wrong) that you'll run into trouble with characters beyond ascii, anything with the high bit set.  Your example in #375 was 0xe4, which is fine in latin-1 (where it represents ä) though notably in utf8, just the byte 0xe4 without  two following byte isn't valid.   But let me know if I'm missing something.  So could you revise the addition to the README to reflect this (or let me know where I'm wrong)

2) Could you merge master back into your branch.  I removed support for python2.6 a while ago but hadn't updated the .travis-ci.yml file because we didn't have travis turned on.  Now that we have travis, I'd like to get the tests passing on all PRs.

Thanks again!

**Comment by jcrowgey** [comment]
One more note, if you wouldn't mind taking a look at whether the man page could use a note as well, I'd appreciate it.  Users struggling with encoding issues on their system may find the man page before the find the README if they install through their package manager.

**Comment by watho** [comment]
Added a simple diagnostics section in manpage for encoding related issues and conky stuff. But it seems to me that the manpage needs a larger reworking. There are most options not documented.

I have merged master and travis is now satisfied :-)

I have to admit i can't follow you on your first point. So far everything works as expected, even with more advanced characters like € and ∑.
I even tried 🎻 (http://www.fileformat.info/info/unicode/char/1f3bb). It's not correctly displayed, because it's not in the font, but for me it is important, that the program runs fine.

File: docs/README.md [hunk 1]
```diff
@@ -483,6 +483,11 @@ ${execpi 300 gcalcli --conky calw 3}
 You may need to increase the `text_buffer_size` in your conkyrc file.  Users
 have reported that the default of 256 bytes is too small for busy calendars.
 
+Additionaly you need to set `--lineart=unicode` to output unicode-characters for box drawing. To avoid misaligned borders use a monospace font like 'DejaVu Sans Mono'. On Python2 it might be necessary to set the environment variable `PYTHONIOENCODING=utf8` if you are using characters beyond 'latin1'. For example:
```
Comment (jcrowgey) [inline docs/README.md]: 'latin1' characters will cause trouble, so this really ought o say 'beyond 'ascii'.

File: docs/man1/gcalcli.1 [hunk 2]
```diff
@@ -98,6 +98,17 @@ gcalcli is a Python application that allows you to access your Google Calendar(s
                            - <mins> default is 10
                            - default command:
                               'notify-send -u critical -i appointment-soon -a gcalcli %%s'
+.SH DIAGNOSTICS
+.B "Encoding issues, wrongly printed characters, UnicodeEncodeError when using conky, in pipes or scripts"
+.RS
```
Comment (jcrowgey) [inline docs/man1/gcalcli.1]: Can we also add a note to try upgrading to python3.

**Pull request merged**